portal/.vscode/extensions.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Are all these  .vscode files recommended as per the angular standards documentation?

Reply:
in angular standard, nothing like this was mentioned, there the document only said that change/modify setting.json files, and these files in vscode was autogenerated and they are also in osmox project so I kept it